### ISSUE TITLE
Image resolution discrepancies for many works that have been synced

### ISSUE THREAD

**Issue by TBRC-Travis** [BuddhistDigitalResourceCenter/archive-issues#17]
During processing I have been using Graphic Converter to rotate images when I see that a particular image is vertical and needs to be made horizontal.  Unfortunately GC adds extra metadata to the image during rotation that causes the image to change resolution to 72ppi.  This messes up how the image is rendered in PDFs.

I am going through all the works I've synced in 2016 and creating a list of images that need to be fixed because of this issue.

**Comment by xristy** [comment]
This is great that you've discovered the a bug in the workflow. Is it possible to run a script exec'ing ImageMagick to extract the resolution metadata and the pixel dimensions of the image and then flag instances where the (dimensions / resolution) are significantly different (via a threshold) from preceeding images. Probably initially just look for 72 dpi since that seems to be what GC is doing to the metadata

**Comment by TBRC-Travis** [comment]
Yes.  That is exactly what I did.  I created a script which runs "identify -verbose | grep "Resolution:"" across all the images and I'll create a little audit script to go through the output and find the 72's that need fixing.
